config.yaml

```diff
@@ -1,15 +1,23 @@
 options:
   allow-privileged:
     type: string
-    default: "auto"
+    default: "false"
```

Comment:
This may be a VERY bad default.  We need `true` here to run CNIs like `calico` and `cilium`.  Perhaps we should leave the default as `true` because otherwise so many things will suddenly start to fail